### PR TITLE
Play 2.5

### PR DESCRIPTION
**Pull request by alexduf**
this update the release tool chain, the scala version and the play version to be compatible with play 2.5.
It is based on @davidfurey's work

The compatibility of the library towards a play version is decided based on the major version number of the library.

### CODE REVIEW

**Comment by alexduf** [comment]
Any comment or objection on this one? @robertberry @rich-nguyen

File: build.sbt [hunk 1]
```diff
@@ -1,27 +1,21 @@
...
-sonatypeSettings
-
 organization := "com.gu"
 
 name := "crosswords-api-client"
 
-scalaVersion := "2.11.4"
-
-crossScalaVersions := Seq("2.10.4", " 2.11.4")
+scalaVersion := "2.11.7"
```
Comment (adamnfish) [inline build.sbt]: Any reason not to use the latest version (2.11.8)?

File: build.sbt [hunk 2]
```diff
@@ -1,27 +1,21 @@
...
   "Typesafe Repository" at "http://repo.typesafe.com/typesafe/releases/"
 )
 
 libraryDependencies ++= Seq(
-  "com.typesafe.play" %% "play-json" % "2.3.7",
-  "joda-time" % "joda-time" % "2.3",
-  "org.specs2" %% "specs2" % "2.3.11" % "test"
+  "com.typesafe.play" %% "play-json" % "2.5.3",
+  "joda-time" % "joda-time" % "2.9.3",
+  "org.specs2" %% "specs2-core" % "3.6" % "test"
```
Comment (adamnfish) [inline build.sbt]: [Looks like there are lots more versions out for this library now](http://notes.implicit.ly/tagged/specs2). That said, maven only shows some of them. Maybe 3.7 is a better choice with that in mind and if that doesn't work there's a 3.3.1 point release.

File: project/plugins.sbt [hunk 3]
```diff
@@ -1,5 +1,5 @@
-addSbtPlugin("com.github.gseitz" % "sbt-release" % "0.8.3")
+addSbtPlugin("com.github.gseitz" % "sbt-release" % "1.0.1")
```
Comment (adamnfish) [inline project/plugins.sbt]: Looks like 1.0.3 is the latest version for this library?

**Comment by adamnfish** [comment]
Not important, but I happened to spot this and that there were newer versions available for some of the libs referenced here.

**Comment by alexduf** [comment]
You're right, I've bumped the libraries. I kept the cross compilation option even though we compile it only for scala 2.11, as in the future we may need scala 2.12

**Comment by rich-nguyen** [comment]
👍  looks good. I do have a branch somewhere for frontend on play 2.5, so grab me if you're looking into that too.

**Pull request merged**